android/src/main/java/com/pdftron/reactnative/views/DocumentView.java

```diff
@@ -1359,13 +1367,10 @@ protected void onAttachedToWindow() {
             setSupportFragmentManager(mFragmentManagerSave);
         }
         // TODO, update base64 when ViewerBuilder supports byte array
-        Uri fileUri = ReactUtils.getUri(getContext(), mDocumentPath, mIsBase64);
+        Uri fileUri = ReactUtils.getUri(getContext(), mDocumentPath, mIsBase64, mBase64Extension);
```

Comment:
here too

Reply:
Are you referring to add a reference to the array? I wonder if this would be duplicated?

Reply:
basically every time you call `getUri` item needs to be added to the array, because that's when the temp file is created, here there are 2 cases, 1 is set tab (when mDocument is null), 2 is change tab (when mDocument is not null)